client/src/app/metadata-schema-form/test-json-files/test-json-schema.json

```diff
@@ -260,6 +260,17 @@
       "type": "array",
       "user_friendly": "INSDC study accession"
     },
+    "ega_accessions": {
```

Comment:
Is this a duplication of the json schema definition? or is it merely for the form rendering?